csp-mixin/panels.libsonnet

```diff
@@ -899,5 +899,667 @@ local commonlib = import 'common-lib/common/main.libsonnet';
+          + g.query.prometheus.withInstant(true)
+          + g.query.prometheus.withFormat('table'),
+        ]
+      )
+      + g.panel.table.standardOptions.withOverrides([
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Time',
+          },
+          properties: [
+            {
+              id: 'custom.hidden',
+              value: true,
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'instance_name',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: 'Instance',
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'job',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: 'Job',
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'project_id',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: 'Project ID',
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Value #tableCpuUtilization',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: 'Cpu utilization',
+            },
+            {
+              id: 'unit',
+              value: 'percent',
+            },
+            {
+              id: 'custom.width',
+              value: 100,
+            },
+            {
+              id: 'custom.cellOptions',
+              value: {
+                mode: 'basic',
+                type: 'gauge',
+                valueDisplayMode: 'text',
+              },
+            },
+            {
+              id: 'thresholds',
+              value: {
+                mode: 'absolute',
+                steps: [
+                  {
+                    color: 'yellow',
+                    value: null,
+                  },
+                  {
+                    color: 'green',
+                    value: 30,
+                  },
+                  {
+                    color: 'red',
+                    value: 85,
+                  },
+                ],
+              },
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Value #tableUptime',
+          },
+          properties: [
+            {
+              id: 'unit',
+              value: 's',
+            },
+            {
+              id: 'custom.cellOptions',
+              value: {
+                type: 'color-text',
+              },
+            },
+            {
+              id: 'color',
+              value: {
+                fixedColor: 'blue',
+                mode: 'fixed',
+              },
+            },
```

Comment:
Any particular reason you show text in blue and no in normal text color (white)? I want to use the same criteria on my dashboard

Reply:
as we use colors on threholds, If you have them in blue it makes me think that blue means something specific and ca be confused

Reply:
I removed this https://github.com/grafana/jsonnet-libs/pull/1336/commits/45514b921f03f2a0266a9b73aaf71372a7400617